tests/home_work.spec.js

```diff
@@ -0,0 +1,36 @@
+import { test, expect } from '@playwright/test';
+   
+     test('Perform Login', async ({ page }) => {
+      await page.goto('https://www.saucedemo.com/');
+      await page.locator('[data-test="username"]').click();
+      await page.locator('[data-test="username"]').fill('standard_user');
+      await page.locator('[data-test="password"]').click();
+      await page.locator('[data-test="password"]').fill('secret_sauce');
+      await page.locator('[data-test="login-button"]').click();
+      const titleLocator = page.locator('[data-test="title"]');
+      await expect(titleLocator).toContainText('Products');
+      const cartIconLocator = page.locator('[data-test="shopping-cart-link"]');
+      await expect(cartIconLocator).toBeVisible();
+      const productCards = page.locator('[data-test="inventory-item"]');
+      const productCount = await productCards.count();
+      expect(productCount).toBeGreaterThan(1);
+    });
+
+  test ('Test 2 - Add product to the cart', async ({ page }) => {
+  await page.goto('https://www.saucedemo.com/');
+  await page.locator('[data-test="username"]').fill('standard_user');
```

Comment:
If u need to login everytime in the test, u can use beforeEach hook to do so.
If you need to login once use beforeAll hook
Please, read more here: https://playwright.dev/docs/api/class-test#test-before-each